uint16 LED0
uint16 LED1
uint16 LED2
uint16 LED3
